gas_v3_doc/readme.md

```diff
@@ -96,14 +96,18 @@ You can also verify that for new account ops, it will cost 2.5Gwei x 40664236 =
   - Basic gas token transfer to existing account: 21000 gas (=0.0001 EOS) (using gas price = 5Gwei)
   - Basic gas token transfer to new account: 20353118 gas (=0.1017 EOS) (It need 40664236 gas to cover the RAM cost in the worst case. Since gas price is 5Gwei, but the storage price is 2.5Gwei, it will discount the gas usage by around 50%)
   - ERC-20 token transfer to existing account: 42620 gas (=0.0002 EOS)
-  - ERC-20 token transfer to new account: 20317594 gas (=0.1017 EOS). (similar reason as basic gas token transfer to new account)
+  - ERC-20 token transfer to new account: 20317594 gas (=0.1016 EOS). (similar reason as basic gas token transfer to new account)
+
+  `gas price` = 5 Gwei (minimum) (requires `gas price` >= max of (`overhead price` & `storage price`)). `inclusion price` = 5Gwei.
+  - ERC-20 token transfer to existing account: 42620 gas (=0.0002 EOS)
```

Comment:
The gas stays the same, but shouldn't the conversion to EOS be double now when including the inclusion price? So approximately 0.0004 EOS instead of 0.0002?